doc/source/setup-jupyterhub.rst

```diff
@@ -64,15 +64,34 @@ For the following steps, use your favorite code editor. We'll use the
 Install JupyterHub
 ------------------
 
-1. Let's use helm to create the instances that you configured with the
-   ``config.yaml`` file. Run this command from the directory that contains the
+1. Let's add the JupyterHub `helm repository<https://github.com/kubernetes/helm/blob/master/docs/chart_repository.md>`_
+   to your helm, so you can install JupyterHub from it. This makes it easy to refer to the JupyterHub chart
+   without having to use a long URL each time.
+
+   .. code:: bash
+
+      helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
```

Comment:
It is the correct url! It redirects you to z2jh if you hit it with a browser. It is just the gh-pages branch of the helm-chart repo, formatted to have the index.yaml file that helm requires.